<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seuillage</class>
 <widget class="QWidget" name="Seuillage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>291</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <item row="5" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="sp_haut">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Type de seuillage : </string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSlider" name="hs_haut">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Seuil bas : </string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Seuil haut : </string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSlider" name="hs_bas">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>128</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Type de seuillage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>automatique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>manuel</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="sp_bas">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>128</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="1" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="boutValider">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hs_bas</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sp_bas</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hs_haut</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>sp_haut</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sp_bas</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hs_bas</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sp_haut</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hs_haut</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
